pkg/reconciler/kubernetes/tektonresult/tektonresult.go

```diff
@@ -147,6 +147,10 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tr *v1alpha1.TektonResul
 	}
 	tr.Status.MarkDependenciesInstalled()
 
+	// This function needs to be removed in the next release of operator.
+	if err := r.conflictTIS(ctx); err != nil {
```

Comment:
why we need to do this before preReconcile?

Reply:
To remove manifest which are common in installerset and pre-installerset.

Reply:
From next release, there won't be anything common in between installerset and pre-installerset.